build.gradle.kts

```diff
@@ -29,4 +30,6 @@ talaiot {
     publishers {
         timelinePublisher = true
     }
-}
\ No newline at end of file
+}
+
+tasks.register("generateConfigOptions") { main() }
```

Comment:
Please rename `main` to a more readable name

Reply:
Renamed to generateConfigOptions()